src/components/@core/navigation-menu/light/left-menu/edit-link-button.tsx

```diff
@@ -15,7 +16,7 @@ export default function EditLinkButton({ label }) {
   const [canEdit, setCanEdit] = useState(false);
 
   useEffect(() => {
-    setCanEdit(adminOrAuthor(authorId) && id && !pathname.endsWith("edit"));
+    setCanEdit(authorId && hasAccess([Role.Admin]) && id && !pathname.endsWith("edit"));
```

Comment:
Seems that this condition is bit wrong, authorId will be true for any user who us logged in and it will always check for admin, If you just want to check for admin then there's no need to check for authorId at all!